config/v1/types_scheduling.go

```diff
@@ -28,20 +28,16 @@ type Scheduler struct {
 }
 
 type SchedulerSpec struct {
-	// DEPRECATED: the scheduler Policy API has been deprecated and will be removed in a future release.
-	// policy is a reference to a ConfigMap containing scheduler policy which has
-	// user specified predicates and priorities. If this ConfigMap is not available
-	// scheduler will default to use DefaultAlgorithmProvider.
-	// The namespace for this configmap is openshift-config.
-	// +optional
-	Policy ConfigMapNameReference `json:"policy,omitempty"`
```

Comment:
Before you can remove a field, you need to understand how many people are using the field, to do so, you'll need to look at [Cluster Fleet Evaluation](https://github.com/openshift/enhancements/blob/1779e1e59e79fac704ac8425d34b3e330e34abf2/dev-guide/cluster-fleet-evaluation.md) and get the telemetry back.

Once you have the telemetry, we can assess how many people are relying on this field, and work out a plan of action to migrate them away from this.

Reply:
So even when a field has been deprecated for many releases we still need to provide evidence the field is no longer used or very rarely?

Reply:
How does that error bubble up? Does it cause the cluster to degrade? How many releases has the error been present for?

Reply:
The kube scheduler operator goes Degraded. For more than 3 releases.

Reply:
I believe the fleet data contains degraded operators, so it would be good still to check the existing fleet data to make sure no one is actually degraded with this. If they are degraded, we may want to reach out.

I think this will still need a staff eng approval for removal, could you please raise a topic on the workloads arch call to discuss the field removal and check that the staff engineers are happy we've done everything we need to do

Reply:
Gotcha. I will open a different PR for the removal of the field. This is currently irrelevant for this PR. Thanks for the clarification.

Reply:
Leaving the removal for another PR.